TurkishFilter/sections/css_extended.txt

```diff
@@ -9,6 +9,8 @@ keyfgazetesi.com#?##siteSag > div[style]:has(> div > ins.adsbygoogle)
 muglayenigun.com#?#section.sidebarWidget.nativeWidget:has(> .textwidget > p > img[src^="https://i.hizliresim.com/"])
 mackolik.com#?#.sidebar > .widget-nesine-most-played-coupons:has(> a[href*="admost.com/adx/"])
 freemagazine.com.tr#?#.solbar > .sol-kutu:has(> .m-baslik > p > b:contains(/^SPONSOR REKLAMLAR$/))
+turkanime.net##div[class^="col-xs-"]:has(> div.panel > div.panel-ust > div.panel-title:contains(SPONSOR REKLAM))
+turkanime.net##div[class^="col-xs-"]:has(> div.panel > div.panel-ust > div.panel-title:contains(FACEBOOK))
```

Comment:
Could you please move this rule to -https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/css_extended.txt